libraries/ui-library/package.json

```diff
@@ -46,11 +46,11 @@
     "@stencil/react-output-target": "^0.5.3",
```

Comment:
Did you check if there is also a newer version available for the different output targets? :)

Reply:
looks like they are already up to date